frontend/pkg/frontend/ocm.go

```diff
@@ -254,3 +258,30 @@ func (f *Frontend) BuildCSNodepool(ctx context.Context, nodepool *api.HCPOpenShi
 
 	return npBuilder.Build()
 }
+
+// GetCSCluster creates and sends a GET request to fetch a cluster from Clusters Service
+func (f *Frontend) GetCSCluster(doc *database.HCPOpenShiftClusterDocument) (*cmv1.ClusterGetResponse, error) {
```

Comment:
```suggestion
func (f *Frontend) GetCSCluster(clusterID string) (*cmv1.ClusterGetResponse, error) {
```
I think this makes more sense/aligns with `DeleteCSCluster` as well - though you'll also need to update wherever this function is used.

Reply:
yeah good call, that makes sense to me too. Fix coming.

Reply:
Updated